tests/Php54/Php54Test.php

```diff
@@ -34,4 +34,14 @@ public function testClassUsesInvalid()
     {
         $this->assertFalse(@class_uses('NotDefined'));
     }
+
+    public function testHexDecode()
+    {
+        $php54 = new \Symfony\Polyfill\Php54\Php54;
+        // issue #48: hex2bin() returns false if nul byte present
+        $this->assertEquals(
+            "\x61\x62\x00\x63\x64",
+            $php54->hex2bin("6162006364")
```

Comment:
Please test against the `hex2bin` function directly. The test suite automatically tests against the polyfill and the native implementations.